chapters/version_control_with_git/chapter.md

```diff
@@ -369,8 +370,7 @@ It can be used to compare states between all kinds of areas (check out the examp
 Let's check it out:
 
 ```bash
-$ git diff
-diff --git a/src/ofApp.cpp b/src/ofApp.cpp
+$ git diff --git a/src/ofApp.cpp b/src/ofApp.cpp
```

Comment:
same here. It was correct before. it is what the`git diff` command will output and not extra arguments for it.